### PR TITLE
6062 migrate user detail part 3

### PR DESCRIPTION
**Pull request by ThibaudAV**
**Issue**

https://github.com/gravitee-io/issues/issues/6062

**Description**

- Bump storybook and active feature to try it. 
> If i understand properly this feature active something like SSR for storybook and make it faster 

- Add environment roles settings un user datails

**Additional context**

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-cgqnnteyui.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6062-migrate-user-detail-part-3/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-ixnlnmfxqq.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-wuhrjucbbo.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6062-migrate-user-detail-part-3/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

File: gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail.component.scss [hunk 1]
```diff
@@ -89,4 +89,14 @@
     display: flex;
     flex-direction: column;
   }
+
+  &__environments-card {
+    &__table {
+      .mat-cell.mat-column-roles {
+        display: flex;
+        flex-direction: column;
+        padding-top: 1.34375em; // to center mat-form-field in cell. Same value of mat-form-field's bottom padding
```
Comment (gaetanmaisse) [inline gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail.component.scss:98]: 🙈

File: gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail.component.ts [hunk 2]
```diff
@@ -112,4 +157,23 @@ export class OrgSettingsUserDetailComponent implements OnInit, OnDestroy {
       this.toggleSaveBar(true);
     });
   }
+
+  private initEnvironmentsRolesForm(environments: Environment[]) {
+    this.environmentsRolesFormGroup = new FormGroup({
+      ...environments.reduce((result, environment) => {
+        const userEnvRoles = this.user.envRoles[environment.id];
+
+        if (!userEnvRoles) return result;
```
Comment (gaetanmaisse) [inline gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail.component.ts]: Not a big fan of statements without `{ ... }`

File: gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail.component.ts [hunk 3]
```diff
@@ -112,4 +157,21 @@ export class OrgSettingsUserDetailComponent implements OnInit, OnDestroy {
       this.toggleSaveBar(true);
     });
   }
+
+  private initEnvironmentsRolesForm(environments: Environment[]) {
+    this.environmentsRolesFormGroup = new FormGroup({
+      ...environments.reduce((result, environment) => {
```
Comment (gaetanmaisse) [inline gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail.component.ts]: Don't need to destructurate, you can use reduce's return as it 
```
new FormGroup(environments.reduce...)
```

**Review by gaetanmaisse — APPROVED** [review]

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-dzdwrcnhzt.chromatic.com)

File: gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail.component.ts (hunk 2 above)
Comment (ThibaudAV) [reply gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail.component.ts]: removed :p

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6062-migrate-user-detail-part-3/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

File: gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail.component.ts (hunk 3 above)
Comment (ThibaudAV) [reply gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail.component.ts]: 🚀

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-accyzvuiww.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6062-migrate-user-detail-part-3/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-qvetunvzmn.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6062-migrate-user-detail-part-3/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Pull request merged**